reconcile/utils/saasherder.py

```diff
@@ -206,14 +218,78 @@ def _validate_saas_files(self):
             for saas_file_name, saas_file_paths in duplicates.items():
                 logging.error(msg.format(saas_file_name, saas_file_paths))
 
-        # promotion publish channel duplicates
-        duplicates = [p for p in saas_file_promotion_publish_channels
-                      if saas_file_promotion_publish_channels.count(p) > 1]
-        if duplicates:
-            self.valid = False
-            msg = 'saas file promotion publish channel is not unique: {}'
-            for duplicate in duplicates:
-                logging.error(msg.format(duplicate))
+        # Promotions have the same source repository
```

Comment:
The comment feels redundant here. The function name describes it accurately already.

Reply:
Commend removed.